dropwizard-logging/src/main/java/io/dropwizard/logging/LoggingFactory.java

```diff
@@ -143,7 +157,7 @@ private void configureInstrumentation(Logger root, MetricRegistry metricRegistry
     }
 
     private Logger configureLevels() {
-        final Logger root = (Logger) LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
+        final Logger root = loggerContext.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
         root.getLoggerContext().reset();
```

Comment:
How about to replace calls to `root.getLoggerContext()` to `loggerContext`, if we have it as a field?